### PR TITLE
Paper: Likeness: a toolkit for connecting the social fabric of place to human dynamics

### PR DESCRIPTION
**Pull request by jGaboardi**
If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.

*Editor*: @deniederhut

*Reviewers*: @asdhob

cc @jvtcl

### CODE REVIEW

**Comment by scoobies** [comment]
Hi!, I'm @scoobies, a friendly bot.

Type ```@scoobies help``` to discover how I can help you.

**Comment by jGaboardi** [comment]
@scoobies build paper

**Comment by scoobies** [comment]
building paper...

**Comment by scoobies** [comment]
{"data"=>{"info"=>"Build for paper jGaboardi-2022_tuccardi scheduled.  Note that builds are only executed if the current build attempt is more than 5 minutes old."}, "status"=>"success"}

**Comment by jGaboardi** [comment]
@scoobies check references

**Comment by scoobies** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/s11116-017-9760-8 is OK
- 10.1109/SC.2008.5214892 is OK
- 10.1177/1460458206061217 is OK
- 10.3390/ijerph14070730 is OK
- 10.1016/j.compenvurbsys.2017.05.004 is OK
- 10.1016/j.eswa.2009.01.060 is OK
- 10.1016/j.cor.2006.10.014 is OK
- 10.1038/sj.bjc.6604462 is OK
- 10.1137/0215016 is OK
- 10.1111/gean.12211 is OK
- 10.5281/zenodo.6335783 is OK
- 10.18564/jasss.1909 is OK
- 10.1038/s41586-020-2649-2 is OK
- 10.1002/sapm1941201224 is OK
- 10.1016/j.vaccine.2011.06.059 is OK
- 10.1109/MCSE.2007.55 is OK
- 10.5281/zenodo.5573592 is OK
- 10.1037/0033-2909.85.2.410 is OK
- 10.2307/1907301 is OK
- 10.1111/gean.12011 is OK
- 10.1016/j.compenvurbsys.2013.03.004 is OK
- 10.25080/Majora-92bf1922-00a is OK
- 10.1111/gec3.12204 is OK
- 10.1111/j.1475-4762.2008.00844.x is OK
- 10.1007/978-3-319-22786-3_5 is OK
- 10.1080/00045608.2013.843439 is OK
- 10.1109/TPAMI.2007.53 is OK
- 10.1371/journal.pone.0056079 is OK
- 10.1111/gean.12016 is OK
- 10.5281/zenodo.3509134 is OK
- 10.48690/1523377 is OK
- 10.1111/j.1524-4733.2008.00437.x is OK
- 10.1016/j.healthplace.2011.01.001 is OK
- 10.18637/jss.v079.i10 is OK
- 10.1145/2820783.2820882 is OK
- 10.1111/j.1475-4762.2008.00837.x is OK
- 10.5281/zenodo.6607533 is OK
- 10.1038/s41592-019-0686-2 is OK
- 10.3768/rtipress.2009.mr.0010.0905 is OK
- 10.3141/2429-18 is OK

MISSING DOIs

- 10.1111/j.1477-9730.2005.00343_3.x may be a valid DOI for title: Geographic Information Systems for Transportation: Principles and Applications

INVALID DOIs

- 10.1016/0965-85649600004-3 is INVALID
- https://doi.org/10.18637/jss.v063.i17 is INVALID because of 'https://doi.org/' prefix
- 10.1016/S2214-109X2030452-6 is INVALID
```

**Comment by jGaboardi** [comment]
@scoobies build paper

**Comment by scoobies** [comment]
building paper...

**Comment by scoobies** [comment]
{"data"=>{"info"=>"Build for paper jGaboardi-2022_tuccardi scheduled.  Note that builds are only executed if the current build attempt is more than 5 minutes old."}, "status"=>"success"}

**Comment by jGaboardi** [comment]
@scoobies check references

**Comment by scoobies** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/s11116-017-9760-8 is OK
- 10.1109/SC.2008.5214892 is OK
- 10.1177/1460458206061217 is OK
- 10.3390/ijerph14070730 is OK
- 10.1016/j.compenvurbsys.2017.05.004 is OK
- 10.1016/j.eswa.2009.01.060 is OK
- 10.1016/j.cor.2006.10.014 is OK
- 10.1038/sj.bjc.6604462 is OK
- 10.1137/0215016 is OK
- 10.1111/gean.12211 is OK
- 10.5281/zenodo.6335783 is OK
- 10.18637/jss.v063.i17 is OK
- 10.18564/jasss.1909 is OK
- 10.1038/s41586-020-2649-2 is OK
- 10.1002/sapm1941201224 is OK
- 10.1016/j.vaccine.2011.06.059 is OK
- 10.1109/MCSE.2007.55 is OK
- 10.5281/zenodo.5573592 is OK
- 10.1037/0033-2909.85.2.410 is OK
- 10.2307/1907301 is OK
- 10.1111/gean.12011 is OK
- 10.1016/j.compenvurbsys.2013.03.004 is OK
- 10.25080/Majora-92bf1922-00a is OK
- 10.1111/gec3.12204 is OK
- 10.1111/j.1475-4762.2008.00844.x is OK
- 10.1007/978-3-319-22786-3_5 is OK
- 10.1080/00045608.2013.843439 is OK
- 10.1109/TPAMI.2007.53 is OK
- 10.1371/journal.pone.0056079 is OK
- 10.1111/gean.12016 is OK
- 10.5281/zenodo.3509134 is OK
- 10.48690/1523377 is OK
- 10.1111/j.1524-4733.2008.00437.x is OK
- 10.1016/j.healthplace.2011.01.001 is OK
- 10.18637/jss.v079.i10 is OK
- 10.1145/2820783.2820882 is OK
- 10.1111/j.1475-4762.2008.00837.x is OK
- 10.5281/zenodo.6607533 is OK
- 10.1038/s41592-019-0686-2 is OK
- 10.3768/rtipress.2009.mr.0010.0905 is OK
- 10.3141/2429-18 is OK

MISSING DOIs

- 10.1111/j.1477-9730.2005.00343_3.x may be a valid DOI for title: Geographic Information Systems for Transportation: Principles and Applications

INVALID DOIs

- 10.1016/0965-85649600004-3 is INVALID
- 10.1016/S2214-109X2030452-6 is INVALID
```

**Comment by jGaboardi** [comment]
xref [#710](https://github.com/scipy-conference/scipy_proceedings/pull/710#issuecomment-1144377890)

I'm also having trouble with Scoobies' DOI checker. The two Invalid DOIs listed above have parentheses in them. The PDF build locally and on the server with properly formatted and functional DOI links, but not with Scoobies.

* `10.1016/0965-8564(96)00004-3`
* `10.1016/S2214-109X(20)30452-6`

**Comment by jGaboardi** [comment]
@deniederhut I am happy to volunteer if you are in need of extra reviewers for proceedings submissions.

**Comment by deniederhut** [comment]
> I'm also having trouble with Scoobies' DOI checker.

Sorry! I'm not sure why scoobies is having trouble, but we can check those ones manually if we need to.

**Comment by deniederhut** [comment]
@scoobies mark paper

**Comment by deniederhut** [comment]
@scoobies mark needs review

**Comment by deniederhut** [comment]
@scoobies assign @deniederhut as editor

**Comment by scoobies** [comment]
Assigned! @deniederhut is now the editor

**Comment by jGaboardi** [comment]
> > I'm also having trouble with Scoobies' DOI checker.
> 
> Sorry! I'm not sure why scoobies is having trouble, but we can check those ones manually if we need to.

Thanks! I am stickler for DOI, myself. All should be in order.

**Comment by deniederhut** [comment]
@scoobies add @ashob to reviewers

**Comment by scoobies** [comment]
@ashob added to the reviewers list!

**Comment by deniederhut** [comment]
Hi! We've moved the deadline for first reviews back one week, to June 16th, to give us a little more time to find reviewers for all the papers this year. You can see the updated full timeline at https://github.com/scipy-conference/scipy_proceedings#timeline-for-2022.

**Comment by lemythe** [comment]
Ok, I accept to review this paper

**Comment by deniederhut** [comment]
@scoobies add @lemythe to reviewers

**Comment by scoobies** [comment]
@lemythe added to the reviewers list!

**Comment by deniederhut** [comment]
Hey @ashob and @lemythe! How are the reviews coming along?

**Comment by jGaboardi** [comment]
@ashob, @lemythe, @deniederhut We have added several more sentences to 2nd to last paragraph in the *Conclusion*.

**Comment by jGaboardi** [comment]
@scoobies check references

**Comment by scoobies** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1007/s11116-017-9760-8 is OK
- 10.1109/SC.2008.5214892 is OK
- 10.1177/1460458206061217 is OK
- 10.3390/ijerph14070730 is OK
- 10.1016/j.compenvurbsys.2017.05.004 is OK
- 10.1016/j.eswa.2009.01.060 is OK
- 10.1016/j.cor.2006.10.014 is OK
- 10.1038/sj.bjc.6604462 is OK
- 10.1111/j.1467-9787.2011.00743.x is OK
- 10.5281/zenodo.4444156 is OK
- 10.21105/joss.03330 is OK
- 10.1137/0215016 is OK
- 10.1111/gean.12211 is OK
- 10.5281/zenodo.6335783 is OK
- 10.18637/jss.v063.i17 is OK
- 10.18564/jasss.1909 is OK
- 10.1038/s41586-020-2649-2 is OK
- 10.1002/sapm1941201224 is OK
- 10.1016/j.vaccine.2011.06.059 is OK
- 10.1109/MCSE.2007.55 is OK
- 10.5281/zenodo.5573592 is OK
- 10.1037/0033-2909.85.2.410 is OK
- 10.2307/1907301 is OK
- 10.1111/gean.12011 is OK
- 10.1016/j.compenvurbsys.2013.03.004 is OK
- 10.25080/Majora-92bf1922-00a is OK
- 10.1111/gec3.12204 is OK
- 10.1111/j.1475-4762.2008.00844.x is OK
- 10.1007/978-3-319-22786-3_5 is OK
- 10.1080/00045608.2013.843439 is OK
- 10.1109/TPAMI.2007.53 is OK
- 10.1371/journal.pone.0056079 is OK
- 10.1111/gean.12016 is OK
- 10.5281/zenodo.3509134 is OK
- 10.52324/001c.8285 is OK
- 10.1111/gean.12276 is OK
- 10.48690/1523377 is OK
- 10.1111/j.1524-4733.2008.00437.x is OK
- 10.1016/j.healthplace.2011.01.001 is OK
- 10.18637/jss.v079.i10 is OK
- 10.1145/2820783.2820882 is OK
- 10.1111/j.1475-4762.2008.00837.x is OK
- 10.5281/zenodo.6607533 is OK
- 10.1038/s41592-019-0686-2 is OK
- 10.3768/rtipress.2009.mr.0010.0905 is OK
- 10.3141/2429-18 is OK

MISSING DOIs

- 10.1111/j.1477-9730.2005.00343_3.x may be a valid DOI for title: Geographic Information Systems for Transportation: Principles and Applications

INVALID DOIs

- 10.1016/0965-85649600004-3 is INVALID
- 10.1016/S2214-109X2030452-6 is INVALID
```

**Comment by jGaboardi** [comment]
@scoobies build paper

**Comment by scoobies** [comment]
building paper...

**Comment by scoobies** [comment]
{"data"=>{"info"=>"Build for paper jGaboardi-2022_tuccardi scheduled.  Note that builds are only executed if the current build attempt is more than 5 minutes old."}, "status"=>"success"}

**Comment by deniederhut** [comment]
@scoobies remove @ashob from reviewers

**Comment by scoobies** [comment]
@ashob removed from the reviewers list!

**Comment by deniederhut** [comment]
@scoobies add @asdhob to reviewers

**Comment by scoobies** [comment]
@asdhob added to the reviewers list!

**Comment by deniederhut** [comment]
Hey @lemythe and @asdhob! Friendly reminder that reviews are due tomorrow

**Comment by asdhob** [comment]
Hi everyone, introducing myself here: my name is Ajit, currently working as a Data Scientist at Capital One. The first Scipy conference I attended was in 2018 !

**Comment by jGaboardi** [comment]
James here. My first SciPy was 2016. This is my first time presenting (a poster) and submitting a proceedings paper. This is my co-author's (@jvtcl) first SciPy. We are both Associate Research Scientists at Oak Ridge National Laboratory and hold PhDs in Geography.

File: papers/james_gaboardi/likeness.rst [hunk 1]
```diff
@@ -0,0 +1,739 @@
...
+often do not capture a wide enough range of individual characteristics to assess how human
+dynamics are linked to human security problems (e.g., how a person's age, limited
+transportation access, and linguistic isolation may interact with their housing situation
+in a flood evacuation emergency).
+
+In this paper, we introduce Likeness :cite:`tuccillo_jospeh_v_2022_6607533`, a Python
+toolkit for connecting the social fabric of place to human dynamics via models that
+support increased spatial, temporal, and demographic fidelity. Likeness is an extension of
+the UrbanPop framework developed at Oak Ridge National Laboratory (ORNL) that embraces a
+new paradigm of "vivid" synthetic populations, in which individual agents may be
```
Comment (asdhob) [inline papers/james_gaboardi/likeness.rst]: Can you provide any prior reference to this? like this one: [from ACS conference](https://acsdatacommunity.prb.org/discussion-forum/m/2021-acs-conference-files/161/download)

File: papers/james_gaboardi/likeness.rst [hunk 2]
```diff
@@ -0,0 +1,739 @@
...
+activity allocation. We provide preliminary usage examples for Likeness based on 1) social
+contact networks in POIs 2) 24-hour POI occupancy characteristics. Finally, we discuss
+existing limitations and outlook for future development.
+
+
+Overview of Core Capabilities and Workflow
+------------------------------------------
+
+UrbanPop initially combined the vivid synthetic populations produced from the American
+Community Survey (ACS) using P-MEDM with a commute model based on origin-destination
```
Comment (asdhob) [inline papers/james_gaboardi/likeness.rst]: Please add the acronym for P-MEDM & MOE during their first appearance.

File: papers/james_gaboardi/likeness.rst [hunk 3]
```diff
@@ -0,0 +1,739 @@
...
+Synthetic populations are typically generated from census microdata, which consists of a
+sample of publicly available longform responses to official statistical surveys. To
+preserve respondent confidentiality, census microdata is often published at spatial
+scales the size of a city or larger. A family of spatial allocation methods known as
+Iterative Proportional Fitting (IPF) provide a maximum-likelihood estimator for microdata
+responses in small (e.g., neighborhood) areas based on aggregate data published about
+those areas (known as "constraints"), resulting in a baseline for population synthesis
+:cite:`wheaton2009synthesized, beckman1996creating, templ2017simulation`. UrbanPop is
+built upon a regularized implementation of IPF, the *Penalized-Maximum Entropy
+Daysmetric Modeling* (P-MEDM) method, that permits many more input census variables than
```
Comment (asdhob) [inline papers/james_gaboardi/likeness.rst]: Please correct the spelling for `Penalized Maximum Entropy Dasymetric Model`.

File: papers/james_gaboardi/likeness.rst [hunk 4]
```diff
@@ -0,0 +1,739 @@
...
+traditional approaches :cite:`leyk_maximum_2013, nagle2014dasymetric`. The P-MEDM
+objective function (Eq. :ref:`eq:pmedm`) is written as: 
+
+.. math::
+   :label: eq:pmedm
+
+   \max -\sum_{it}\frac{n}{N}\frac{w_{it}}{d_{it}}\log\frac{w_{it}}{d_{it}} -\sum_{k}\frac{e^2_{k}}{2\sigma_{k}^2}
```
Comment (asdhob) [inline papers/james_gaboardi/likeness.rst:178]: Add information/definition for the 2nd summation term (& its variables e, σ): e^2/2*σ^2

File: papers/james_gaboardi/likeness.rst [hunk 5]
```diff
@@ -0,0 +1,739 @@
...
+of individuals matching responses from the ACS Public-Use Microdata Sample (PUMS) at the
+scale of census block groups (typically 300–6000 people) or tracts (1200–8000 people),
+depending upon the use-case.
+
+Downscaling the PUMS from the Public-Use Microdata Area (PUMA) level at which it is
+offered (100,000 or more people) to these neighborhood scales then enables us to produce
+synthetic populations (the ``livelike`` package) and simulate their travel to POIs
+(the ``actlike`` package) in an integrated model. To our knowledge, this integrated
+model is the first instance of a microsimulation that is carried out with respect to both
+real-world transportation networks and POIs.
```
Comment (asdhob) [inline papers/james_gaboardi/likeness.rst]: Is this claim too strong?

File: papers/james_gaboardi/likeness.rst [hunk 6]
```diff
@@ -0,0 +1,739 @@
...
+- Performed *IBB* allocation to simulate the household locations of workers and students. Our selection of household locations for workers and students varied geographically. Because school attendance in Knox County is restricted by district boundaries, we only placed student households in the PUMAs intersecting with the district (FIPS 4701601, 4701602, 4701603, 4701604). However, because educators may live outside school district boundaries, we simulated their household locations throughout the Knoxville CBSA.
+- Used ``actlike`` to perform optimal allocation of workers and students about road networks in Knox County/Knoxville CBSA. Across the 30 simulations and 14 segments identified, we produced a total of 420 travel simulations. Network impedance was measured in length for all students simulations and travel time for all educator simulations.
```
Comment (asdhob) [inline papers/james_gaboardi/likeness.rst]: So `length` here can be clarified as the geographic distance?

File: papers/james_gaboardi/likeness.rst [hunk 7]
```diff
@@ -0,0 +1,739 @@
...
+
+   Optimal allocations for one simulation of 10\ :sup:`th` grade public schools in Knox County, TN. :label:`fig:k12:10pub:student`
+
+.. _Students:
+
+Students
+++++++++
+
+Our study of K–12 students examine social contact networks with respect potentially
+underserved student populations via the compositional characteristics of POIs (schools).
```
Comment (asdhob) [inline papers/james_gaboardi/likeness.rst]: Correction: with respect `to` potentially underserved student populations.

File: papers/james_gaboardi/likeness.rst [hunk 8]
```diff
@@ -0,0 +1,739 @@
...
+A determination of modeling output robustness is needed to validate our results.
+Specifically, we aimed to ensure the preservation of relative facility size and
+composition. To perform this validation, we tested the optimal allocations of those
+generated by Likeness against the maximally adjusted reported enrollment & faculty
+employment counts. We used the maximum adjusted value to account for scenarios where the
+population synthesis phase resulted in a total demographic segment greater than reported
+total facility capacity. We employed Canonical Correlation Analysis (CCA)
+:cite:`knapp_canonical_1978` for the K–12 public school student allocations due to their
+stratified nature, and an ordinary least squares (OLS) simple linear regression for the
+educator allocations :cite:`scikit-learn`. The CCA for students was performed in two components:
```
Comment (asdhob) [inline papers/james_gaboardi/likeness.rst]: Please provide the motivation behind usage of OLS linear regression for the educator allocations.

**Review by asdhob — COMMENTED** [review]
In this paper, the authors showcase the Likeness toolkit, which can be used for  understanding where the population lives, how they travel and make social interactions. The authors have tried to make the toolkit generalized, so that it is feasible for usage in different applications and fields. 
Clarity is provided through different examples described in the paper: examples of the different population segments, applications like transportation, emergency situations like flood. It clears the understanding for a new reader on human dynamics, activity spaces and their interaction. 
The visualizations are captivating and directly address the case study, and the interpretation from them are really insightful. Limitations for the case study have been considered appropriately, including the changing life patterns due to Covid.
The case study chosen is paramount in light of recent school shootings. Understanding how student and teacher populations commute, interact and their geospatial features, has never been more important, especially for improving the emergency response from law enforcement agencies. 
It’s a very exciting project, in terms of what the future holds for this, regarding various applications in different fields. Looking forward to the open-sourcing of Likeness’ code base this year.

I've added a few recommendations/questions through individual comments.

**Comment by jGaboardi** [comment]
@asdhob Thanks so much for the review. @jvtcl and I will get to editing and responding to suggestions promptly.

**Comment by deniederhut** [comment]
@scoobies mark needs more review

File: papers/james_gaboardi/likeness.rst (hunk 2 above)
Comment (jGaboardi) [reply papers/james_gaboardi/likeness.rst]: resolved via b7de1ee and 6e2a110

File: papers/james_gaboardi/likeness.rst (hunk 3 above)
Comment (jGaboardi) [reply papers/james_gaboardi/likeness.rst]: resolved via af867f8

File: papers/james_gaboardi/likeness.rst (hunk 6 above)
Comment (jGaboardi) [reply papers/james_gaboardi/likeness.rst]: resolved via 3cc49bd

File: papers/james_gaboardi/likeness.rst (hunk 7 above)
Comment (jGaboardi) [reply papers/james_gaboardi/likeness.rst]: resolved via 6681777

**Comment by deniederhut** [comment]
@scoobies remove @lemythe from reviewers

**Comment by scoobies** [comment]
@lemythe removed from the reviewers list!

**Comment by deniederhut** [comment]
@scoobies mark pending comment

File: papers/james_gaboardi/likeness.rst (hunk 1 above)
Comment (jGaboardi) [reply papers/james_gaboardi/likeness.rst]: Added 2 references for this with b15d593.

File: papers/james_gaboardi/likeness.rst (hunk 4 above)
Comment (jGaboardi) [reply papers/james_gaboardi/likeness.rst:178]: Added explanation for the term and variables in b15d593.

File: papers/james_gaboardi/likeness.rst (hunk 8 above)
Comment (jGaboardi) [reply papers/james_gaboardi/likeness.rst]: Explanation provided in 068806d.

File: papers/james_gaboardi/likeness.rst (hunk 5 above)
Comment (jGaboardi) [reply papers/james_gaboardi/likeness.rst]: Good point on this. We reworded for a less "definitive" statement (see 068806d).

**Comment by jGaboardi** [comment]
@asdhob We have responded to your suggestions and made the appropriate edits within the manuscript. Thank you for raising these points, they have absolutely improved the quality of our paper.

@deniederhut Along with our edits based on @asdhob's comments, I have given the manuscript a "final" comb through (on our part) for typos, grammar, etc. We believe it is all in order for your review (pending @asdhob's approval).

cc @jvtcl

**Comment by jGaboardi** [comment]
@deniederhut We have a question about proceedings paper acceptance, publication, and distribution. @jvtcl and I would like to provide a QR code in the *poster* we are presenting that links to this paper (assuming it is accepted, etc.). Is this acceptable?

**Comment by deniederhut** [comment]
Yeah that's totally fine. I'm not sure if we'll have the website up before you go to print your poster though? The url should be something like https://conference.scipy.org/proceedings/scipy2022/james_gaboardi.html -- I can do a quick test compile to double check if you like.

**Comment by deniederhut** [comment]
@asdhob do you feel this paper is now ready for inclusion in the proceedings?

**Comment by asdhob** [comment]
Thank you for updating the paper! I've reviewed the changes and the paper looks ready for the proceedings.

**Comment by jGaboardi** [comment]
> Yeah that's totally fine. I'm not sure if we'll have the website up before you go to print your poster though? The url should be something like https://conference.scipy.org/proceedings/scipy2022/james_gaboardi.html -- I can do a quick test compile to double check if you like.

Yeah, that'd be awesome!

**Comment by deniederhut** [comment]
Okay yeah that's the right URL.

**Review by deniederhut — APPROVED** [review]
Finally had a chance to read through the paper -- holy cow this is cool!

**Comment by deniederhut** [comment]
@scoobies mark ready

**Comment by jGaboardi** [comment]
> Okay yeah that's the right URL.

@deniederhut Two questions:
* Can triple check that the [https://conference.scipy.org/proceedings/scipy2022/james_gaboardi.html] URL will work (and be stable) once the proceedings are published?
* Will that link work if we use it in the QR code in our poster? I just realized the [Timeline](https://github.com/scipy-conference/scipy_proceedings#timeline-for-2022) says that the window is July 30th.

cc @jvtcl

**Comment by deniederhut** [comment]
Yup so we publish the proceedings twice -- once the week before the conference (we're aiming for July 4th for this year) and then a finalized version with any additional posters/slides/typo-fixes-that-people-notice-after-the-first-publication after the conference (usually the end of July). The URL is based on the name of the folder with your paper in it, so that shouldn't change.

**Comment by jGaboardi** [comment]
> Yup so we publish the proceedings twice -- once the week before the conference (we're aiming for July 4th for this year) and then a finalized version with any additional posters/slides/typo-fixes-that-people-notice-after-the-first-publication after the conference (usually the end of July). The URL is based on the name of the folder with your paper in it, so that shouldn't change.

Sweet! Thanks so much for putting up with my 8 million questions!

We are finalizing our poster this week and we will be presenting it in person. But we should open a PR for it in here, as well, correct?

**Comment by deniederhut** [comment]
> Thanks so much for putting up with my 8 million questions!

I only counted 7.5? So I think you still have some questions left 😄

> But we should open a PR for it in here, as well, correct?

If you want to, sure, but it's not required. We archive them on zenodo and put a link to them in the proceedings -- here's what it looks like in the html version (from last year) https://conference.scipy.org/proceedings/scipy2021/slides.html

**Pull request merged**